app/liquid/liquid_renderer.rb

```diff
@@ -134,6 +134,7 @@ def isolate_from_plugin_data(field)
 
   def location
     return @location if @location.blank?
+    return {} if @location.data['country_code'] == 'RD'
```

Comment:
the problem with this, and with the line above, is that we don't send the member's stored country code depending on the geocoder (which gets overridden by the member's stored country code). That doesn't really make sense. Can we change this to make the `@member.country` the first check and send that if it's available?